app.py

```diff
@@ -4,9 +4,29 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
+@app.route('/index', methods=['Get', 'Put'])
```

Comment:
The root (`/`) route has been replaced by `/index`

This will need to be fixed before this is merged in